src/main/java/com/qaprosoft/carina/demo/utils/ScreenshotUtil.java

```diff
@@ -17,32 +19,35 @@
 import java.util.GregorianCalendar;
 
 public class ScreenshotUtil implements IDriverPool {
+    String screenName;
+
+    File outputFile = null;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
     public static Calendar calendar = new GregorianCalendar();
+
     public static SimpleDateFormat sdf = new SimpleDateFormat("MM.dd.yyyy-HH.mm");
 
     public static String dateToString(Date date) {
```

Comment:
you've declared calendar and sdf on 28 and 30 lines. maybe let's create a method to get current date? 
```
public String getCurrentDateAsString() {
       Calendar calendar = new GregorianCalendar();
       SimpleDateFormat sdf = new SimpleDateFormat("MM.dd.yyyy-HH.mm");
       return sdf.format(calendar.getTIme());
}
```